airbyte-integrations/connectors/source-facebook-marketing/unit_tests/test_config_migrations.py

```diff
@@ -23,10 +24,14 @@ def load_config(config_path: str) -> Mapping[str, Any]:
         return json.load(config)
 
 
+def _config_path(path_from_unit_test_folder: str) -> str:
+    return str(pathlib.Path(__file__).parent / path_from_unit_test_folder)
+
+
 class TestMigrateAccountIdToArray:
-    TEST_CONFIG_PATH = "unit_tests/test_migrations/account_id_to_array/test_old_config.json"
-    NEW_TEST_CONFIG_PATH = "unit_tests/test_migrations/account_id_to_array/test_new_config.json"
-    UPGRADED_TEST_CONFIG_PATH = "unit_tests/test_migrations/account_id_to_array/test_upgraded_config.json"
+    TEST_CONFIG_PATH = _config_path("test_migrations/account_id_to_array/test_old_config.json")
```

Comment:
the paths here and below have long prefixes which are the same maybe you can build the full path with a variable with a short name. Something like: `_config_path(f"{prefix}/test_new_config.json"` and so on. Just less text.